test/auth/auth.test.js

```diff
@@ -0,0 +1,62 @@
+/* eslint-env mocha */
+import proclaim from 'proclaim';
+import fetchMock from 'fetch-mock';
+
+import {getJsonWebToken} from '../..//src/js/utils/auth';
+
+describe("Auth", () => {
+	describe("getJsonWebToken", () => {
+		it("is a function", () => {
+			proclaim.isFunction(getJsonWebToken);
+		});
+
+		describe("when comments auth service returns a valid response", () => {
+			before(() => {
+				fetchMock.mock('https://comments-auth.ft.com/v1/jwt/', {
+					token: '12345'
+				});
+			});
+
+			after(() => {
+				fetchMock.reset();
+			});
+
+			it("returns a promise", () => {
+				const returnValue = getJsonWebToken();
+				proclaim.isInstanceOf(returnValue, Promise);
+			});
+
+			it("returns a promise which contains a JSON Web Token as a string", () => {
+				getJsonWebToken()
```

Comment:
This test may be giving a false positive.
```suggestion
				return getJsonWebToken()
```